<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MesonRewriterOptionContainer</class>
 <widget class="QWidget" name="MesonRewriterOptionContainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>32</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="h_layout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="b_delete">
     <property name="icon">
      <iconset theme="edit-delete"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b_delete</sender>
   <signal>clicked()</signal>
   <receiver>MesonRewriterOptionContainer</receiver>
   <slot>deleteMe()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deleteMe()</slot>
 </slots>
</ui>
